src/PollService.js

```diff
@@ -0,0 +1,75 @@
+import { PrivateService } from '@makerdao/services-core';
+
+const mockParsedAllPollsData = [
+  {
+    creator: '0xeda95d1bdb60f901986f43459151b6d1c734b8a2',
+    pollId: 0,
+    blockCreated: 123456789,
+    startTime: new Date(),
+    endTime: new Date(),
+    multiHash: 'QmaozNR7DZHQK1ZcU9p7QdrshMvXqWK6gpu5rmrkPdT3L4'
+  }
```

Comment:
@tyler17 Would the poll address/source be returned with this data as well?

Reply:
@b-pmcg What do you mean by poll address/source?
cc @jparklev

Reply:
@b-pmcg do the designs suggest there should be a poll address? It was put forward at one point that each poll be an individual contract (rather than an event); we ended up not going down that path, but i was wondering the designs were totally up-to-date on that.

Reply:
@jparklev I do see a field for `source` on the mockups for the new new polling page. I wonder if that should be changed to the hash/voteId?

https://www.figma.com/file/P3WU8leDECFDItgLx4gh22t9/Governance?node-id=3631%3A181